app/models/user.rb

```diff
@@ -9,6 +9,8 @@ class User < ApplicationRecord
   validates :last_name, presence: true
   validates :email, presence: true
 
+  audited
```

Comment:
The ticket also includes adding this to the `trainee` and `provider` models.

Reply:
(Ah, but just seen your comment on the PR!)

Reply:
I'll do the other models too 👌